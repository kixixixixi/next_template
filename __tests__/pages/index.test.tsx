import React from "react"
import renderer from "react-test-renderer"
import IndexPage from "../../pages/index"
describe("App", () => {
  it("renders correctly", async () => {
    const component = renderer.create(<IndexPage />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
