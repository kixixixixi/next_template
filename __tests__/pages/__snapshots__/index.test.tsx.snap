// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<main>
  <section>
    <h1>
      Hello, world!
    </h1>
  </section>
</main>
`;
